src/Mailtrap.php

```diff
@@ -131,10 +151,46 @@ public function fetchAttachmentsOfLastMessage()
     {
         $email = $this->fetchLastMessage();
         $response = $this->client->get("inboxes/{$this->config['inbox_id']}/messages/{$email['id']}/attachments")->getBody();
-
         return json_decode($response, true);
     }
 
+    public function fetchAttachmentsOfMessage($inboxID, $messageID)
+    {
+        if ($inboxID != ''){ // test sent a specific email box to search
+            $messages = $this->client->get("inboxes/$inboxID/messages/$messageID/attachments")->getBody();
+            $messages = json_decode($messages, true);           
+       } else { // Use the config email box
+            $messages = $this->client->get("inboxes/{$this->config['inbox_id']}/messages/$messageID/attachments")->getBody();
+            $messages = json_decode($messages, true);
+        }
+        return array_shift($messages);
+    }
+
```

Comment:
- The else statement could be avoided here as well
- The `$inboxID` should be the second parameter and optional as it will default to the config.